lib/discourse_theme/cli.rb

```diff
@@ -200,6 +205,41 @@ def run(args)
 
     private
 
+    def ignored_migrations(theme, dir)
+      return [] unless theme && Dir.exist?(File.join(dir, "migrations"))
+
+      existing_migrations =
+        theme
+          .dig("theme_fields")
+          &.filter_map do |theme_field|
+            theme_field["name"] if theme_field["target"] == "migrations"
+          end || []
+
+      new_migrations =
+        Dir["#{dir}/migrations/**/*.js"].reject do |f|
+          existing_migrations.any? do |existing_migration|
+            File.basename(f).include?(existing_migration)
+          end
+        end
+
+      if !new_migrations.empty?
+        options = { "Yes" => :yes, "No" => :no }
+
+        choice = UI.select(<<~TEXT, options.keys)
+        Would you like to upload and run the following pending theme migration(s): #{new_migrations.join(", ")}
```

Comment:
From the PR description it seems like `new_migrations` here prints absolute paths of the pending migrations, can we instead print relative paths? I don't think there's any value in printing the full paths and they make the message longer and harder to read.

Reply:
Fixed :)